tools/deploy.py

```diff
@@ -1,7 +1,11 @@
+# Standard
 import getpass
+import grp
 import logging
 import os
 import shutil
+
+# Cheese shop
```

Comment:
I think the comments where stuff comes from are not necessary (at least I don't use them and never saw them used in other projects). It's also not correct here: pathlib is in the standard library since at least 3.4.

Reply:
Ah, yes. I'm a little old-skool (working daytime on 2.7 ATM). The comments are likely a work-place induced 'habit' damage :).